translations/strings.yaml

```diff
@@ -244,9 +244,18 @@ customDNSSettings:
 networkSettings:
   multihopTitle: Multi-hop tunnel
 
+
+captivePortalAlert:
+  title: Captive portal detected!
+  header: A captive portal is when the guest Wi-Fi you are using requires you to log in to get internet access. 
+  actionPreActivation: You can turn on your VPN once you have gained internet access by logging in to the portal. 
+  actionPostActivation: You must turn off your VPN in order to see the portal, and turn it back on once you have logged in. 
+  buttonTextPreActivation: Okay
+  buttonTextPostActivation: Turn off VPN
+  
 notifications:
   captivePortalBlockTitle: Guest Wi-Fi portal blocked
-  captivePortalBlockMessage: The guest Wi-Fi network you’re connected to requires action. Click here to turn off VPN to see the portal.
+  captivePortalBlockMessage2: The guest Wi-Fi network you’re connected to requires action. Click to turn off VPN to see the portal.
```

Comment:
I mentioned this before: is click the right word for these? This codebase support desktop AND mobile devices.

Reply:
@chrisrooneys